.php_cs.dist

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Rules we follow are from PSR-2 as well as the rectified PSR-2 guide.
+ *
+ * - https://github.com/FriendsOfPHP/PHP-CS-Fixer
+ * - https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
+ * - https://github.com/php-fig-rectified/fig-rectified-standards/blob/master/PSR-2-R-coding-style-guide-additions.md
+ *
+ * If something isn't addressed in either of those, some other common community rules are
+ * used that might not be addressed explicitly in PSR-2 in order to improve code quality
+ * (so that devs don't need to comment on them in Code Reviews).
+ *
+ * For instance: removing trailing white space, removing extra line breaks where
+ * they're not needed (back to back, beginning or end of function/class, etc.),
+ * adding trailing commas in the last line of an array, etc.
+ */
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude('node_modules')
+    ->exclude('vendor')
+    ->in(__DIR__);
+
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@PSR2' => true,
```

Comment:
So. I was wondering since it said `@PSR2` at the top, why the other rules?

I removed them all and ran `fix` and it did quite some changes, like these:
```diff
diff --git a/src/GraphQL.php b/src/GraphQL.php
index c5d3b0d..9562554 100644
--- a/src/GraphQL.php
+++ b/src/GraphQL.php
@@ -239,10 +239,12 @@ class GraphQL

         if (! $type instanceof TypeConvertible) {
             throw new TypeNotFound(
-                sprintf('Unable to convert %s to a GraphQL type, please add/implement the interface %s',
+                sprintf(
+                    'Unable to convert %s to a GraphQL type, please add/implement the interface %s',
                     get_class($type),
                     TypeConvertible::class
-                ));
+                )
+            );
         }

         foreach ($opts as $key => $value) {
diff --git a/src/Support/ResolveInfoFieldsAndArguments.php b/src/Support/ResolveInfoFieldsAndArguments.php
index 34aceea..bed6194 100644
--- a/src/Support/ResolveInfoFieldsAndArguments.php
+++ b/src/Support/ResolveInfoFieldsAndArguments.php
@@ -130,12 +130,16 @@ class ResolveInfoFieldsAndArguments
                 $spreadName = $selectionNode->name->value;
                 if (isset($this->info->fragments[$spreadName])) {
                     $fragment = $this->info->fragments[$spreadName];
-                    $fields = array_merge_recursive($this->foldSelectionSet($fragment->selectionSet, $descend),
-                        $fields);
+                    $fields = array_merge_recursive(
+                        $this->foldSelectionSet($fragment->selectionSet, $descend),
+                        $fields
+                    );
                 }
             } elseif ($selectionNode instanceof InlineFragmentNode) {
-                $fields = array_merge_recursive($this->foldSelectionSet($selectionNode->selectionSet, $descend),
-                    $fields);
+                $fields = array_merge_recursive(
+                    $this->foldSelectionSet($selectionNode->selectionSet, $descend),
+                    $fields
+                );
             }
         }
```
or
```diff
diff --git a/tests/Database/MutationValidationUniqueWithCustomRulesTests/MutationValidationUniqueWithCustomRulesTest.php b/tests/Database/MutationValidationUniqueWithCustomRulesTests/MutationValidationUniqueWithCustomRulesTest.php
index 82a2932..e1a7581 100644
--- a/tests/Database/MutationValidationUniqueWithCustomRulesTests/MutationValidationUniqueWithCustomRulesTest.php
+++ b/tests/Database/MutationValidationUniqueWithCustomRulesTests/MutationValidationUniqueWithCustomRulesTest.php
@@ -36,7 +36,8 @@ GRAPHQL;
             ],
         ]);

-        $this->assertSqlQueries(<<<'SQL'
+        $this->assertSqlQueries(
+            <<<'SQL'
```
Basically it felt more aggressive.

I liked that because for example, in the _current_ version these two patterns are allowed:
```php
        $this->assertSqlQueries(
            <<<'SQL'
```
and
```php
        $this->assertSqlQueries(<<<'SQL'
```
Nether one is touched and converted to the other.

Yet when I remove all the settings, it's converted the former form always.

I think this is better, the source will be more unified and leaves less room for interpretation.